activestorage/test/service/azure_storage_service_test.rb

```diff
@@ -9,6 +9,27 @@ class ActiveStorage::Service::AzureStorageServiceTest < ActiveSupport::TestCase
 
     include ActiveStorage::Service::SharedServiceTests
 
+    test "direct upload with content disposition" do
```

Comment:
Can this be a shared service test? ☝️

Reply:
I don't think that can be done because the way the content disposition is checked is different for each service.